hackathon_site/registration/models.py

```diff
@@ -99,21 +121,83 @@ class Application(models.Model):
         ],
         null=False,
     )
-    q1 = models.TextField(null=False, help_text="First question?", max_length=1000)
-    q2 = models.TextField(null=False, help_text="Second question?", max_length=1000)
-    q3 = models.TextField(null=False, help_text="Third question?", max_length=1000)
+
+    address1 = models.CharField(
+        max_length=255, null=True, blank=True, help_text="e.g. 35 St George St"
+    )
+    address2 = models.CharField(
+        max_length=255, null=True, blank=True, help_text="e.g. Apt. No. 13"
+    )
+    country = models.CharField(max_length=255, null=False, blank=False)
+    city = models.CharField(max_length=255, null=True, blank=True)
+    region = models.CharField(max_length=255, null=True, blank=True)
+    postal_code = models.CharField(max_length=6, null=True, blank=True)
+
+    how_many_hackathons = models.TextField(
+        null=False,
+        help_text="How many hackathons have you been to?",
+        choices=HACKATHON_NUMBER_CHOICES,
+        max_length=100,
+    )
+
+    what_hackathon_experience = models.TextField(
+        null=False,
+        help_text="If you’ve been to a hackathon, briefly tell us your experience. If not, describe what you expect to see and experience.",
+        max_length=1000,
+    )
+
+    why_participate = models.TextField(
+        null=False,
+        help_text="Why do you want to participate in NewHacks?",
+        max_length=1000,
+    )
+
+    what_technical_experience = models.TextField(
+        null=False,
+        help_text="What is your technical experience with software and hardware?",
+        max_length=1000,
+    )
+
+    hardware_preference = models.CharField(
+        null=False,
+        help_text="Please indicate your preference for using hardware in this hackathon",
```

Comment:
I don't see this sentence on the form? Is it for the radio button? Is the help text intentionally omitted on the form?